scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1122,7 +1236,7 @@ public boolean linkResourceManager(String rmURL) throws NotConnectedException, P
     @Override
     public boolean reloadPolicyConfiguration() throws NotConnectedException, PermissionException {
         frontendState.checkPermission("reloadPolicyConfiguration",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)